app/tests/benchmarks.c

```diff
@@ -158,12 +158,14 @@ __NO_INLINE static void arm_bench_cset_stm(void)
 __NO_INLINE static void arm_bench_multi_issue(void)
 {
     uint32_t cycles;
+#if       (__CORTEX_M >= 0x03)
```

Comment:
may as well just #ifdef out this entire function, since there's no point trying to benchmark multi-issue on a non multi-issue cpu.